### PR TITLE
Tests infrastructure. Some test cases

### PR DESCRIPTION
**Pull request by cyberstar**
Introduces basic tox setup and helper application for test cases (models and admin).

Described models needed for most of test cases.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+source = aldryn_reversion
+omit = *migrations*
+
+[report]
+exclude_lines =
+    pragma: no cover
+    def __repr__
+    raise AssertionError
+    raise NotImplementedError
+    if __name__ == .__main__.:
```
Comment (intellectronica) [inline .coveragerc]: newline

**Comment by intellectronica** [comment]
I don't think the source passes PEP8, can you please check?

**Comment by intellectronica** [comment]
but other than that LGTM

**Comment by cyberstar** [comment]
will be split in smaller PRs. Tests infrastructure is in PR #11

**Pull request closed without merging**